hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/ClientExceptionsUtil.java

```diff
@@ -87,15 +90,15 @@ public static Throwable findException(Object exception) {
     }
     Throwable cur = (Throwable) exception;
     while (cur != null) {
-      if (isSpecialException(cur)) {
+      if (matchExceptionWeCare(cur)) {
```

Comment:
nit: although not imp, can you keep the name same as before? 
`matchExceptionWeCare` doesn't seem much vocal or may be we don't need that one liner function.

Reply:
The isSpecialException has been removed so we need a new method. And regionDefinatelyOnRegionServer is not a good name here?

Reply:
Agree it should have meaningful name. I was thinking rather than `matchExceptionWeCare` which might not be a generic method nor meaningful, is it fine to write this directly? :
```
if(regionDefinitelyOnTheRegionServerException(cur) || regionServerIsOverloadException(cur) || cur instanceof NotServingRegionException)
```